helmfile.d/values/networkpolicies/service/thanos.yaml.gotmpl

```
@@ -87,6 +87,9 @@ policies:
         - rule: egress-rule-dns
         - rule: egress-rule-query
       ingress:
+        - rule: ingress-rule-apiserver
```

Please add a comment that explains why this network policy is needed, something like:
```suggestion
        # This is needed to allow the diagnostic script proxying Thanos.
        - rule: ingress-rule-apiserver
```